src/coord/info_command.c

```diff
@@ -329,7 +329,7 @@ static void replyKvArray(RedisModule_Reply *reply, InfoFields *fields, InfoValue
   }
 }
 
-static void generateFieldsReply(InfoFields *fields, RedisModule_Reply *reply) {
+static void generateFieldsReply(InfoFields *fields, RedisModule_Reply *reply, bool obfuscate) {
```

Comment:
So the coordinator simply aggregates the replies it gets and forwards them.
It is more of a question if the shard obfuscated the data or not.

Looked at the shard level and found some fields that can be hidden, will make the required changes regarding them.

Going back to the coordinator, since both the coordinator and shard use `IndexError_Reply` the bool got propagated up in the coordinator.
I can remove the boolean from the function.
WDYT?